ChartboostAdapter/src/main/java/com/chartboost/mediation/chartboostadapter/ChartboostAdapter.kt

```diff
@@ -327,6 +335,7 @@ class ChartboostAdapter : PartnerAdapter {
                 PartnerLogController.log(INVALIDATE_SUCCEEDED)
                 Result.success(partnerAd)
             }
+            else -> Result.failure(ChartboostMediationAdException(ChartboostMediationError.CM_LOAD_FAILURE_UNSUPPORTED_AD_FORMAT))
```

Comment:
Granted we don't have one for invalidate for unsupported ad format yet so let's use another invalidate failure one.

Reply:
oh shoot I meant to make all of these succeed